File: SecondYear/29deepanshutyagi_DeepanshuTyagi_2226cs1195_2/Week_3/maximumandminimum.cpp
#include <iostream>
using namespace std;
pair<long long, long long> getMinMax(long long a[], int n) {
     long long minVal = LLONG_MAX;
    long long maxVal = LLONG_MIN;

    for (int i = 0; i < n; i++) {
        minVal = min(minVal, a[i]);
        maxVal = max(maxVal, a[i]);
    }

    return make_pair(minVal, maxVal);
}

File: SecondYear/aayushraghav93_AayushRaghav_2226CSE(AI)1108_2/Week_3/maximumandminimum.java
class Compute 
{
    static Pair getMinMax(long a[], long n)  
    {
        //Write your code here
        long min=Integer.MAX_VALUE,max=Integer.MIN_VALUE;
        for(int i=0;i<n;i++){
            if(a[i]>max)
            max=a[i];
            if(a[i]<min)
            min=a[i];
        }
        Pair minmax=new Pair(min,max);
        return minmax;
    }
}
